### PR TITLE
Publish YDB CLI 2.18.0

### PR DESCRIPTION
**Pull request by asmyasnikov**
### Changelog entry 

Publish YDB CLI 2.18.0 with Release Notes

### Changelog category 

* Documentation (changelog entry is not required)

### CODE REVIEW

File: ydb/docs/en/core/changelog-cli.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 # {{ ydb-short-name }} CLI changelog
 
+## Version 2.18.0 {#2-18-0}
+
+Released on December 24, 2024. To update to version **2.18.0**, select the [Downloads](downloads/index.md#ydb-cli) section.
+
+* Query plan and statistics improvements:
+  * Expression and attributes added to various operator properties (i.e. GroupBy)
+  * Per operator statistics (Rows)
+  * Statistics from Column Shards (Rows and Bytes)
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:10]:
```suggestion
* Query plan and statistics provide additional information:
  * Expression and attributes added to various operator properties (e.g., `GroupBy`)
  * Per-operator statistics (Rows)
  * Statistics for [column-oriented tables](./concepts/datamodel/table.md#column-oriented-tables) (Rows and Bytes)
```

Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:10]: Clarify what those Rows and Bytes mean. Probably rows processed for operators and rows/bytes read from column shards, but I had to guess that.

File: ydb/docs/en/core/changelog-cli.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+* Fixed a bug where `ydb workload * run` command could crash in `--dry-run` mode.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:11]:
```suggestion
* Fixed a bug where the `ydb workload * run` command could crash in `--dry-run` mode.
```

File: ydb/docs/en/core/changelog-cli.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+* Added support for views in local backups: `ydb tools dump` and `ydb tools restore`. Views are backed up as `CREATE VIEW` queries saved in the `create_view.sql` files, which can be executed to recreate the original views.
+* Replaced option `--query-settings` by `--query-prefix` one in `ydb workload <workload> run`.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:13]:
```suggestion
* Added support for views in local backups: `ydb tools dump` and `ydb tools restore`. Views are backed up as `CREATE VIEW` queries saved in the `create_view.sql` files, which can be executed to recreate the original views.
* Replaced the `--query-settings` option with `--query-prefix` in `ydb workload <workload> run`.
```

File: ydb/docs/en/core/changelog-cli.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
+## Version 2.18.0 {#2-18-0}
+
+Released on December 24, 2024. To update to version **2.18.0**, select the [Downloads](downloads/index.md#ydb-cli) section.
+
+* Query plan and statistics improvements:
+  * Expression and attributes added to various operator properties (i.e. GroupBy)
+  * Per operator statistics (Rows)
+  * Statistics from Column Shards (Rows and Bytes)
+* Fixed a bug where `ydb workload * run` command could crash in `--dry-run` mode.
+* Added support for views in local backups: `ydb tools dump` and `ydb tools restore`. Views are backed up as `CREATE VIEW` queries saved in the `create_view.sql` files, which can be executed to recreate the original views.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:12]: Was there a PR adding this to the actual documentation for these commands?

File: ydb/docs/en/core/changelog-cli.md (hunk 3 above)
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:13]: This needs to be marked as a backward-incompatible change if it doesn't support the old name anymore.

File: ydb/docs/en/core/changelog-cli.md [hunk 5]
```diff
@@ -1,5 +1,26 @@
...
+* Replaced option `--query-settings` by `--query-prefix` one in `ydb workload <workload> run`.
+* Added new options to `ydb workload topic`: --tx-commit-interval and --tx-commit-messages, allowing you to specify commit interval either in milliseconds or in number of messages written.
+Also now you can load test YDB topics, using wide transactions that span over all partitions in the topic. This works both in write and in end-to-end workload scenarios.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:15]:
```suggestion
* Added new options to `ydb workload topic`: `--tx-commit-interval` and `--tx-commit-messages`, allowing you to specify the commit interval in milliseconds or by the number of messages written.
* Added support for load testing [{{ ydb-short-name }} topics](./concepts/topic.md) using wide transactions that span all partitions in the topic. This works in both write and end-to-end workload scenarios.
```

Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:15]: Needs a link to documentation on how to run this workload.

File: ydb/docs/en/core/changelog-cli.md [hunk 6]
```diff
@@ -1,5 +1,26 @@
...
+* `ydb import file csv` command now saves import progress. Relaunching import command will continue from the line it was interrupted on
+* Use QueryService by default (`--executer generic`) in `ydb workload kv` and `ydb workload stock` commands
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:17]:
```suggestion
* The `ydb import file csv` command now saves import progress. Relaunching the import command will continue from the row where it was interrupted.
* In the `ydb workload kv` and `ydb workload stock` commands, the default value of the `--executer` option has been changed to `generic`, which makes them no longer rely on the legacy query execution infrastructure.
```

File: ydb/docs/en/core/changelog-cli.md [hunk 7]
```diff
@@ -1,5 +1,26 @@
...
+* Use parquet format instead of CSV to fill tables in `ydb workload` benchmarks
+* Made `--consumer` flag in `ydb topic read` command optional. Now if this flag is not specified, reading is performed in no-consumer mode. In this mode partition IDs should be specified with `--partition-ids` option.
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:19]:
```suggestion
* Replaced the CSV format with Parquet for filling tables in `ydb workload` benchmarks.
* Made the `--consumer` flag in the `ydb topic read` command optional. If this flag is not specified, reading is performed in no-consumer mode. In this mode, partition IDs must be specified with the `--partition-ids` option.
```

File: ydb/docs/en/core/changelog-cli.md [hunk 8]
```diff
@@ -1,5 +1,26 @@
...
+* Fixed a bug in `ydb import file csv` where multiple columns with escaped quotes in the same row were parsed incorrectly
+* Truncate query results output in benchmarks
+* Added `ydb admin cluster bootstrap` command to bootstrap automatically configured cluster
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:22]:
```suggestion
* Fixed a bug in `ydb import file csv` where multiple columns with escaped quotes in the same row were parsed incorrectly.
* Truncated query results output in benchmarks.
* Added the `ydb admin cluster bootstrap` command to bootstrap an automatically configured cluster.
```

**Review by blinkov — DISMISSED** [review]

File: ydb/docs/en/core/changelog-cli.md (hunk 8 above)
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:22]: Afaik, this command would work only with a server built from main or 25.1+

Comment (galnat) [reply ydb/docs/en/core/changelog-cli.md]: Саша Рутковский говорит, что должно уже сейчас работать.

**Review by anton-bobkov — DISMISSED** [review]

File: ydb/docs/ru/core/changelog-cli.md [hunk 9]
```diff
@@ -2,6 +2,29 @@
...
+* В [команду](./reference/ydb-cli/workload-topic#run-write) `ydb workload topic run` добавлены опции `--tx-commit-interval` и `--tx-commit-messages`, которые задают интервал между коммитами транзакций в миллисекундах и в количестве записанных сообщений соответственно.
+* В [команде](./reference/ydb-cli/topic-read) `ydb topic read` параметр `--consumer` перестал быть обязательным. В режиме чтения без подписчика обязательно должны быть указаны идентификаторы партиций с помощью параметра `--partition-ids`. Чтение в этом случае выполняется без сохранения коммита оффсетов.
+* [Команда](./reference/ydb-cli/export-import/import-file) `ydb import file csv` теперь сохраняет прогресс выполнения. Повторный запуск команды импорта продолжится с той строки, на которой она была прервана.
+* В командах `ydb workload kv` и `ydb workload stock` значение параметра `--executer` по умолчанию изменено на "generic", благодаря чему они больше не используют устаревшую инфраструктуру выполнения запросов.
+* Изменен формат загрузки данных в таблицы для нагрузочных тестов `ydb workload` с CSV на Parquet.
+* Добавлена команда `ydb admin cluster bootstrap` для автоматической первоначальной настройки кластера.
```
Comment (fomichev3000) [inline ydb/docs/ru/core/changelog-cli.md:17]: Она не работает. Я бы исключил...

**Review by fomichev3000 — CHANGES_REQUESTED** [review]

File: ydb/docs/ru/core/changelog-cli.md (hunk 9 above)
Comment (galnat) [reply ydb/docs/ru/core/changelog-cli.md]: Убрала

**Review by fomichev3000 — APPROVED** [review]

**Review by pnv1 — APPROVED** [review]

**Pull request merged**